tests/tests/parsers/code_22e9.log

```diff
@@ -0,0 +1,3 @@
+2024-10-15T12:32:31.521795 000 RQ --- 18:130140 32:022222 --:------ 22E9 001 00         # {}
+2024-10-15T12:32:31.538027 082 RP --- 32:022222 18:130140 --:------ 22E9 004 00010000   # {'percent_2': 0.005, 'percent_4': 0.0, 'percent_6': 0.0}      # Vasco D60
+2024-10-15T22:59:02.967986 084 RP --- 32:022222 18:130140 --:------ 22E9 004 00010701   # {'percent_2': 0.005, 'percent_4': 0.035, 'percent_6': 0.005}  # idem
```

Comment:
```suggestion
2024-10-15T12:32:31.521795 000 RQ --- 18:130140 32:022222 --:------ 22E9 001 00        # {}
2024-10-15T12:32:31.538027 082 RP --- 32:022222 18:130140 --:------ 22E9 004 00010000  # {'percent_2': 0.005, 'percent_4': 0.0, 'percent_6': 0.0}      # Vasco D60
2024-10-15T22:59:02.967986 084 RP --- 32:022222 18:130140 --:------ 22E9 004 00010701  # {'percent_2': 0.005, 'percent_4': 0.035, 'percent_6': 0.005}  # idem
```

Reply:
Generally, exactly two spaces before the `#`

Reply:
Command `22E9` is not well understood - the payload is speculative.

Perhaps you could add some context (via comments) in the log file? That is, what was the system doing when it sent these packets - what insights do you have as to their purpose?

Reply:
Will have to focus on that, as it is an RP. When I have something running via ramses_cc I can match the D60 state with less effort